charts/aws-ebs-csi-driver/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 1.2.0
 name: aws-ebs-csi-driver
 description: A Helm chart for AWS EBS CSI Driver
-version: 2.0.4
+version: 2.0.5
```

Comment:
This PR is adding new features to the chart.  The version should be 2.1.0

Reply:
I wasn't sure, a lot of PRs I've been doing recently have only be accepted as patches, will fix.

Reply:
Done.